app/services/BusinessMatchingService.scala

```diff
@@ -36,4 +36,14 @@ class BusinessMatchingService @Inject()(businessMatchingConnector: BusinessMatch
     }
   }
 
+  def sendBusinessMatchingInformation(userAnswers: UserAnswers)
+                                     (implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Option[HttpResponse]] = {
+
+    (userAnswers.get(UniqueTaxpayerReferencePage), BusinessMatchingSubmission(userAnswers)) match {
+      case (Some(utr), Some(organisationSubmission)) =>
+        businessMatchingConnector.sendBusinessMatchingInformation(utr, organisationSubmission).map(Some(_))
```

Comment:
pull down the conversion to the model into the service, I probably should have done that in the individual matching above. Then you have a Future[Option[BusinessAddress]] and in the controller you can recover from the future failed and show the error screen. I'll refactor the individual call once this is merged.